src/compiler/parser-decl-symbtype.bas

```diff
@@ -505,25 +503,26 @@ end function
 
 '':::::
 ''SymbolType      =   CONST? UNSIGNED? (
-''				      ANY
```

Comment:
OK, so tabs have been replaced with spaces for alignment.   I understand.  Except, the comment is not aligned.  Even though these are whitespace changes, still need to review and check that they are correct.  What's the point of the change if needs to changed again?